frontend/src/containers/OnBoarding.js

```diff
@@ -42,58 +54,55 @@ export class OnBoarding extends Component {
     const {
       githubUsername,
       fetchReposFromGitHub,
-      appendGithubForm
+      appendGithubForm,
+      params
     } = this.props;
 
     if (githubUsername && this.state.step === 2) {
-      appendGithubForm({username: githubUsername});
+      appendGithubForm({username: githubUsername, token: params.token});
       fetchReposFromGitHub(githubUsername);
     }
   }
 
   render() {
     const { step } = this.state;
-    const { contributors, githubForm } = this.props;
-    let { repositories } = this.props;
-
-    repositories = repositories.filter((repo) => repo.stars >= MIN_STARS_FOR_ONBOARDING);
-    repositories.sort((A, B) => B.stars - A.stars);
+    const { contributors, githubForm, repositories } = this.props;
 
     return (
       <div className={`OnBoarding ${step ? '-registering' : ''}`}>
         <Notification />
         {step !== 5 && <OnBoardingHeader active={Boolean(step)} username={githubForm.attributes.username} />}
         {step === 0 && <OnBoardingHero onClickStart={() => this.setState({step: 1})} />}
         {step === 1 && <OnBoardingStepConnectGithub />}
-        {step === 2 && <OnBoardingStepPickRepository repositories={repositories} onNextStep={(selectedRepo) => this.getContributors.bind(this, selectedRepo)} />}
-        {step === 3 && <OnBoardingStepPickCoreContributors contributors={contributors} onNextStep={(chosenContributors) => this.setState({step: 4, chosenContributors})} />}
+        {step === 2 && <OnBoardingStepPickRepository repositories={repositories} onNextStep={() => this.getContributors(githubForm.attributes.repository)} {...this.props} />}
+        {step === 3 && <OnBoardingStepPickCoreContributors contributors={contributors} onNextStep={() => this.setState({step: 4})} {...this.props} />}
         {step === 4 && <OnBoardingStepCreate onCreate={this.create.bind(this)} {...this.props} />}
         {step === 5 && <OnBoardingStepThankYou onContinue={() => window.location = '/opensource' }/>}
       </div>
     )
   }
 
-  create(missionDescription, expenseDescription, logo) {
-    const {
-      githubUsername,
-      params,
-      createGroupFromGithubRepo } = this.props;
-    const { selectedRepo, chosenContributors } = this.state;
+  create() {
+    const { githubForm, validateSchema, createGroupFromGithubRepo } = this.props;
 
+    console.log(githubForm.attributes)
+    const attr = githubForm.attributes;
     const payload = {
       group: {
-        name: selectedRepo,
-        slug: selectedRepo,
-        mission: missionDescription,
-        expensePolicy: expenseDescription,
-        logo
+        name: attr.repository,
+        slug: attr.selectedRepo,
```

Comment:
selectedRepo is a param name the attribute property name is `repository` now

should be
```
slug: attr.repository
```